### PR TITLE
Added myself as an author and simple weather app

### PR DESCRIPTION
**Pull request by hjerpbakk**
## Submission description

This is an educational app showcasing a GraphQL Weather API consumed by a Xamarin Forms app. This repository can be used as a template for a modern Xamarin Forms 4.3 app with clean and testable architecture consuming a GraphQL endpoint.

Screens:

https://raw.githubusercontent.com/Sankra/GraphQLDotNet/master/screens.PNG

### CODE REVIEW

File: src/Snppts/Snppts.csproj [hunk 1]
```diff
@@ -6,7 +6,10 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.App" />
```
Comment (jfversluis) [inline src/Snppts/Snppts.csproj]: I don't think this was intentional? Would you mind undoing this change? It's probably nothing, but I would hate if this messes with our CI/CD pipeline somehow ;)

File: src/Snppts/Snippets/SimplestWeather.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+        public bool ContainsAndroidSample => true;
+
+        public bool ContainsiOSSample => true;
+
+        public bool ContainsUWPSample => false;
+
+        public IAmAnAuthor AuthorInfo => new RunarOvesenHjerpbakk();
+
+        public IEnumerable<Uri> ImageUris => new [] { new Uri("https://raw.githubusercontent.com/Sankra/GraphQLDotNet/master/screens.PNG") };
```
Comment (jfversluis) [inline src/Snppts/Snippets/SimplestWeather.cs:28]: Not sure how this image will behave in our layout, but let's see 😄

**Review by jfversluis — CHANGES_REQUESTED** [review]
Undo the csproj change and I think we're all good!

File: src/Snppts/Snppts.csproj (hunk 1 above)
Comment (hjerpbakk) [reply src/Snppts/Snppts.csproj]: Done, must've been some VS magic.

**Review by jfversluis — APPROVED** [review]
Thanks for adding this great contribution!

**Pull request merged**